addons/website_slides/static/src/js/slides_share.js

```diff
@@ -101,14 +65,11 @@ sAnimations.registry.websiteSlidesShare = sAnimations.Class.extend({
      * @param {Object} ev
      */
     _onSlidesSocialShare: function (ev) {
-        var self = this;
         ev.preventDefault();
-        var key = $(ev.currentTarget).attr('social-key');
         var popUpURL = $(ev.currentTarget).attr('href');
         var popUp = window.open(popUpURL, 'Share Dialog', 'width=626,height=436');
         $(window).on('focus', function () {
```

Comment:
@tde-banana-odoo The whole `$(window).on('focus', ...` thing could have been removed IMO.
Too late :p